components/molecules/m-product-additional-info.vue

```diff
@@ -117,10 +120,6 @@ export default {
   }
 }
 
-[itemprop="description"] > *:first-child {
```

Comment:
what was this CSS doing? what changes happen by removing it?

Reply:
Description has 2 paragraphs so according to this CSS margin-top of first paragraph i.e. the first child should be 0. 
But I checked this style and margin wasn't set to 0. It was 16px only. So I decided to remove it.

![itemdesciption](https://user-images.githubusercontent.com/82817616/118808700-66585600-b8c7-11eb-9f45-2970e176f61d.png)

![itemdescription2](https://user-images.githubusercontent.com/82817616/118808781-796b2600-b8c7-11eb-8c2a-201eda2294c0.png)